daq_utils.py

```diff
@@ -5,6 +5,7 @@
 import requests
 import getpass
 import logging
+import warnings
```

Comment:
is there any reason why you aren't using the standard logger?

Reply:
removed